components/lvgl_esp32_drivers/lvgl_spi_conf.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file lvgl_spi_conf.h
+ *
+ */
+
+#ifndef LVGL_SPI_CONF_H
+#define LVGL_SPI_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+/*********************
+ *      DEFINES
+ *********************/
+// DISPLAY PINS
+#define DISP_SPI_MOSI CONFIG_LVGL_DISP_SPI_MOSI
+#define DISP_SPI_CLK CONFIG_LVGL_DISP_SPI_CLK
+#define DISP_SPI_CS CONFIG_LVGL_DISP_SPI_CS
```

Comment:
This symbols are already defined in `disp_spi.h`, can we use those?

Reply:
I thought i removed those from`lvgl_tft/disp_spi.h`. I like that all pins are defined in one file (for instance if you don't want to use `menuconfig` any more)